src/hedit.cpp

```diff
@@ -307,8 +301,7 @@ void hedit_parse(struct descriptor_data *d, const char *arg)
         send_to_char("Writing host to disk.\r\n", d->character);
         write_host_to_disk(d->character->player_specials->saved.zonenum);
         send_to_char("Saved.\r\n", CH);
-        clear_host(d->edit_host);
-        delete d->edit_host;
+        Mem->DeleteHost(d->edit_host);
```

Comment:
What's the difference between clear_host() and DeleteHost? Are we sure that all strings, trigger steps, etc have been copied and that deletion of d->edit_host and all pointed-to memory won't cause issues?

Reply:
Ahh, I see. So the choice of clear_host was deliberately made instead of an oversight? I was thinking using clear_host would have been a source of memory leaks since things were being left behind.